unit_test/live_model_test.cc

```diff
@@ -822,11 +823,15 @@ BOOST_AUTO_TEST_CASE(slates_explore_only_mode) {
   auto it = response.begin();
   size_t action_id = 0;
   auto& slot_response = *it;
+  std::string first_slot_id = slot_response.get_id();
+  BOOST_CHECK_NE(slot_response.get_id(), ""); //uuid
```

Comment:
is there a way to test uuid?

Reply:
I don't know of any, so the best I could think of is to check that the first autogenerated id is not the same as the second autogenerated one